lib/components/narrative/default/default-itinerary.js

```diff
@@ -173,7 +179,11 @@ const ITINERARY_ATTRIBUTES = [
         // FIXME: For CAR mode, walk time considers driving time.
         <>
           <FormattedDuration
-            duration={itinerary.walkTime}
+            duration={
+              itinerary.walkTime > 0
+                ? ensureAtLeastOneMinute(itinerary.walkTime)
+                : itinerary.walkTime
+            }
```

Comment:
Should this condition also appear on line 95?

Reply:
My thinking was it would be nearly impossible to have a leg/itinerary with an exact duration of 0 so it felt like an unnecessary check. However for itineraries that might not have any walk legs, we don't want to accidentally add a default walk time of 1min to an itinerary with absolutely no walking.

Although maybe when you're taking transit it's equally as unlikely to have itineraries with 0 walking, so we should just apply the check across the board to be safe?

Reply:
Leg with duration 0 is very common! That's how we identify transfer legs

Reply:
I think I'd prefer applying the check inside core-utils. Are we ok to block this PR?

Reply:
First of all, it's not how we identify transfer legs, we identify them by checking if `leg.to.stopId === leg.from.stopId`
Second of all, we don't show the leg duration on transfer legs: 
![image](https://github.com/user-attachments/assets/7c6748a9-429c-4cb0-a7e0-0fd3cd3e456d)
So I don't understand why this would be a reason to block the PR

Reply:
Sorry I wasn't clear. The reason to block is to avoid having to check for explicitly 0 leg durations in multiple places

Reply:
I do not think we should have to check for explicitly 0 leg durations in mulitple places. It makes sense to have this check here, because we could feasibly get an itinerary with a 0 walkDuration and we would not want to replace that with a 1min. For example, you plan a trip that starts at one bus stop and ends at another bus stop.

Are there examples where we would need to show a 0min duration for a leg in the itinerary? A leg is going to take you from point A to point B and that's going to take time. The point of the util is to avoid implying that it's going to take 0 time.

Reply:
And just to be clear so I'm not contradicting myself, I know I said 
> so we should just apply the check across the board to be safe?
But thinking about it further, I don't think there's a reason to.